purchase_stock_bl_date/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+* Go to the purchase
+* Create new quotation or edit existing quotation
+* Add bl_date field under Other Information tab.
```

Comment:
This doesn't provide sufficient information.

@rinaldifirdaus Please help improve this.

Reply:
Adjust the usage.rst on the new commit.
please kindly your review.